lib/DebugProcess.php

```diff
@@ -14,6 +14,11 @@ public function __construct(Logger $logger, Bootstrapper $bootstrapper = null) {
     }
 
     protected function doStart(Console $console): \Generator {
+        if ($console->isArgDefined("restart")) {
+            $this->logger->critical("You cannot restart a debug aerys instance via command");
```

Comment:
Sure it is but does it matter for the user? It only matters that the sources get reloaded..?